modules/@apostrophecms/widget-type/ui/apos/components/AposWidgetEditor.vue

```diff
@@ -227,16 +287,39 @@ export default {
           });
           return;
         }
-        if (!widget.type) {
-          widget.type = this.type;
-        }
-        if (!this.id) {
-          widget._id = createId();
-        }
+        this.saving = true;
```

Comment:
Same, would be nice to instantiate this inside data.

Reply:
Same issue, it doesn't need the reactivity.